### DISCUSSION TITLE
How to change profile image?

### DISCUSSION THREAD

**Discussion by shulsher** [community/community#23290] [New to GitHub]
<p>Hi, how to change my profile image? thanks! :slight_smile:</p>

**Comment by wordsoup** [comment, marked as answer]
<p>Hi <a class="mention" href="/shulsher">@shulsher</a>,</p>
<p>Thanks for asking! We have a guide <a href="https://help.github.com/en/github/setting-up-and-managing-your-github-profile/personalizing-your-profile#changing-your-profile-picture" title="Changing your profile picture" rel="nofollow noopener">here</a> to updating your profile picture, including size and dimension guidelines, on our GitHub Help site.</p>
